### PR TITLE
Update create-custom-sensitive-information-types-with-exact-data-matc…

### PR DESCRIPTION
**Pull request by msbemba**
…h-based-classification.md

added a note about the throttling limit for uploading  data into the same data store using EDM Agent.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2370

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md [hunk 1]
```diff
@@ -347,6 +347,10 @@ Proceed to set up your process and schedule for [Refreshing your sensitive info
 
 At this point, you are ready to use EDM-based classification with your Microsoft cloud services. For example, you can [set up a DLP policy using EDM-based classification](#to-create-a-dlp-policy-with-edm).
 
+      > [!NOTE]
+      > The Throttling limit to upload data into the same data store using an EDM agent is set to 2 per day.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md]:
```suggestion
> The throttling limit to upload data into the same data store using an EDM agent is set to two per day.
```

File: microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md [hunk 2]
```diff
@@ -347,6 +347,10 @@ Proceed to set up your process and schedule for [Refreshing your sensitive info
 
 At this point, you are ready to use EDM-based classification with your Microsoft cloud services. For example, you can [set up a DLP policy using EDM-based classification](#to-create-a-dlp-policy-with-edm).
 
+      > [!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-custom-sensitive-information-types-with-exact-data-match-based-classification.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks.

**Comment by chrfox** [comment]
rewording and placing in more appropriate location through an internal PR

**Pull request closed without merging**